src/mongodb/mongo_repository.py

```diff
@@ -1,143 +1,69 @@
 # mongodb/mongo_repository.py
 
-from typing import Dict, Any, Optional, List
-from datetime import datetime
-
-from mongoengine import Q
-
-from src.dal.dal import LibraryItemRepository, PatronRepository, BillRepository
+from typing import List, Optional
 from src.mongodb.mongodb_models.library_item_model import LibraryItemDocument
+from typing import Dict
 from src.mongodb.mongodb_models.patron_model import PatronDocument
 from src.mongodb.mongodb_models.bills_model import BillDocument
 
 
-class MongoLibraryItemRepository(LibraryItemRepository):
-    def update_item_status(self, isbn: str, is_borrowed: bool) -> None:
-        """
-        Update the status of a library item identified by its ISBN.
-
-        Args:
-            isbn (str): The ISBN of the library item.
-            is_borrowed (bool): True if the item is borrowed, False if it is returned.
-        """
-        item = LibraryItemDocument.objects(isbn=isbn).first()
+class MongoLibraryItemRepository:
```

Comment:
Are these classes really needed?
Why not do this functionally

Reply:
i use it because i got the ABC DAL